diff -Nur gnumeric-0.72.orig/Makefile.am gnumeric-0.72/Makefile.am
--- gnumeric-0.72.orig/Makefile.am	Tue Oct 23 15:41:44 2001
+++ gnumeric-0.72/Makefile.am	Tue Oct 23 15:51:21 2001
@@ -20,16 +20,17 @@
 Applicationsdir = $(datadir)/gnome/apps/Applications
 Applications_in_files = gnumeric.desktop.in
 Applications_DATA = gnumeric.desktop
-@XML_I18N_MERGE_DESKTOP_RULE@
 gnewdir = $(datadir)/mc/templates
 gnew_in_files = gnumeric.desktop.in
 gnew_DATA = gnumeric.desktop
-@XML_I18N_MERGE_DESKTOP_RULE@
+@XML_I18N_MERGE_DESKTOP_RULE1@
+@XML_I18N_MERGE_DESKTOP_RULE2@
 
 Mimedir = $(datadir)/mime-info
 Mime_DATA = gnumeric.keys gnumeric.mime
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 oafdir = $(datadir)/oaf
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
diff -Nur gnumeric-0.72.orig/plugins/applix/Makefile.am gnumeric-0.72/plugins/applix/Makefile.am
--- gnumeric-0.72.orig/plugins/applix/Makefile.am	Sun Oct  7 22:59:10 2001
+++ gnumeric-0.72/plugins/applix/Makefile.am	Tue Oct 23 15:48:39 2001
@@ -13,6 +13,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/derivatives/Makefile.am gnumeric-0.72/plugins/derivatives/Makefile.am
--- gnumeric-0.72.orig/plugins/derivatives/Makefile.am	Sun Oct  7 22:59:10 2001
+++ gnumeric-0.72/plugins/derivatives/Makefile.am	Tue Oct 23 15:50:15 2001
@@ -13,6 +13,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/dif/Makefile.am gnumeric-0.72/plugins/dif/Makefile.am
--- gnumeric-0.72.orig/plugins/dif/Makefile.am	Sun Oct  7 22:59:11 2001
+++ gnumeric-0.72/plugins/dif/Makefile.am	Tue Oct 23 15:48:06 2001
@@ -13,6 +13,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/excel/Makefile.am gnumeric-0.72/plugins/excel/Makefile.am
--- gnumeric-0.72.orig/plugins/excel/Makefile.am	Sun Oct  7 22:59:11 2001
+++ gnumeric-0.72/plugins/excel/Makefile.am	Tue Oct 23 15:46:30 2001
@@ -43,6 +43,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/gb/Makefile.am gnumeric-0.72/plugins/gb/Makefile.am
--- gnumeric-0.72.orig/plugins/gb/Makefile.am	Wed Oct 10 20:45:24 2001
+++ gnumeric-0.72/plugins/gb/Makefile.am	Tue Oct 23 15:49:56 2001
@@ -40,6 +40,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = README main.gba gnumeric.gbp $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/gda/Makefile.am gnumeric-0.72/plugins/gda/Makefile.am
--- gnumeric-0.72.orig/plugins/gda/Makefile.am	Sun Oct  7 22:59:12 2001
+++ gnumeric-0.72/plugins/gda/Makefile.am	Tue Oct 23 15:50:24 2001
@@ -15,6 +15,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/gnome-glossary/Makefile.am gnumeric-0.72/plugins/gnome-glossary/Makefile.am
--- gnumeric-0.72.orig/plugins/gnome-glossary/Makefile.am	Tue Aug 21 23:47:24 2001
+++ gnumeric-0.72/plugins/gnome-glossary/Makefile.am	Tue Oct 23 15:49:02 2001
@@ -7,7 +7,8 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(gnumeric_plugin_gnome_glossary_DATA) \
              $(gnumeric_plugin_gnome_glossary_SCRIPTS) \
diff -Nur gnumeric-0.72.orig/plugins/guile/Makefile.am gnumeric-0.72/plugins/guile/Makefile.am
--- gnumeric-0.72.orig/plugins/guile/Makefile.am	Sun Oct  7 22:59:12 2001
+++ gnumeric-0.72/plugins/guile/Makefile.am	Tue Oct 23 15:49:44 2001
@@ -19,6 +19,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(Guilescripts_DATA) $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/html/Makefile.am gnumeric-0.72/plugins/html/Makefile.am
--- gnumeric-0.72.orig/plugins/html/Makefile.am	Sun Oct  7 22:59:12 2001
+++ gnumeric-0.72/plugins/html/Makefile.am	Tue Oct 23 15:47:54 2001
@@ -20,6 +20,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = README ChangeLog $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/lotus-123/Makefile.am gnumeric-0.72/plugins/lotus-123/Makefile.am
--- gnumeric-0.72.orig/plugins/lotus-123/Makefile.am	Sun Oct  7 22:59:13 2001
+++ gnumeric-0.72/plugins/lotus-123/Makefile.am	Tue Oct 23 15:46:40 2001
@@ -19,6 +19,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/mps/Makefile.am gnumeric-0.72/plugins/mps/Makefile.am
--- gnumeric-0.72.orig/plugins/mps/Makefile.am	Sun Oct  7 22:59:15 2001
+++ gnumeric-0.72/plugins/mps/Makefile.am	Tue Oct 23 15:48:50 2001
@@ -13,6 +13,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/numtheory/Makefile.am gnumeric-0.72/plugins/numtheory/Makefile.am
--- gnumeric-0.72.orig/plugins/numtheory/Makefile.am	Sun Oct  7 22:59:16 2001
+++ gnumeric-0.72/plugins/numtheory/Makefile.am	Tue Oct 23 15:50:06 2001
@@ -13,6 +13,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/oleo/Makefile.am gnumeric-0.72/plugins/oleo/Makefile.am
--- gnumeric-0.72.orig/plugins/oleo/Makefile.am	Sun Oct  7 22:59:17 2001
+++ gnumeric-0.72/plugins/oleo/Makefile.am	Tue Oct 23 15:47:10 2001
@@ -13,6 +13,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/perl/Makefile.am gnumeric-0.72/plugins/perl/Makefile.am
--- gnumeric-0.72.orig/plugins/perl/Makefile.am	Mon Oct  8 07:22:11 2001
+++ gnumeric-0.72/plugins/perl/Makefile.am	Tue Oct 23 15:46:16 2001
@@ -74,6 +74,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(Perlscripts_DATA) $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/plan-perfect/Makefile.am gnumeric-0.72/plugins/plan-perfect/Makefile.am
--- gnumeric-0.72.orig/plugins/plan-perfect/Makefile.am	Mon Oct  8 07:22:11 2001
+++ gnumeric-0.72/plugins/plan-perfect/Makefile.am	Tue Oct 23 15:48:17 2001
@@ -22,6 +22,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/psiconv/Makefile.am gnumeric-0.72/plugins/psiconv/Makefile.am
--- gnumeric-0.72.orig/plugins/psiconv/Makefile.am	Sun Oct  7 22:59:18 2001
+++ gnumeric-0.72/plugins/psiconv/Makefile.am	Tue Oct 23 15:49:14 2001
@@ -15,6 +15,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/py-func/Makefile.am gnumeric-0.72/plugins/py-func/Makefile.am
--- gnumeric-0.72.orig/plugins/py-func/Makefile.am	Tue Aug 21 23:47:30 2001
+++ gnumeric-0.72/plugins/py-func/Makefile.am	Tue Oct 23 15:50:36 2001
@@ -6,6 +6,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(gnumeric_plugin_py_func_SCRIPTS) $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/python/Makefile.am gnumeric-0.72/plugins/python/Makefile.am
--- gnumeric-0.72.orig/plugins/python/Makefile.am	Sun Oct  7 22:59:19 2001
+++ gnumeric-0.72/plugins/python/Makefile.am	Tue Oct 23 15:49:31 2001
@@ -20,6 +20,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(Pythonscripts_DATA) $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/python-loader/Makefile.am gnumeric-0.72/plugins/python-loader/Makefile.am
--- gnumeric-0.72.orig/plugins/python-loader/Makefile.am	Sun Oct  7 22:59:19 2001
+++ gnumeric-0.72/plugins/python-loader/Makefile.am	Tue Oct 23 15:50:48 2001
@@ -18,6 +18,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/sc/Makefile.am gnumeric-0.72/plugins/sc/Makefile.am
--- gnumeric-0.72.orig/plugins/sc/Makefile.am	Sun Oct  7 22:59:19 2001
+++ gnumeric-0.72/plugins/sc/Makefile.am	Tue Oct 23 15:47:23 2001
@@ -13,6 +13,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/sylk/Makefile.am gnumeric-0.72/plugins/sylk/Makefile.am
--- gnumeric-0.72.orig/plugins/sylk/Makefile.am	Sun Oct  7 22:59:20 2001
+++ gnumeric-0.72/plugins/sylk/Makefile.am	Tue Oct 23 15:47:34 2001
@@ -13,6 +13,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/xbase/Makefile.am gnumeric-0.72/plugins/xbase/Makefile.am
--- gnumeric-0.72.orig/plugins/xbase/Makefile.am	Sun Oct  7 22:59:20 2001
+++ gnumeric-0.72/plugins/xbase/Makefile.am	Tue Oct 23 15:47:44 2001
@@ -13,6 +13,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
diff -Nur gnumeric-0.72.orig/plugins/xml_sax/Makefile.am gnumeric-0.72/plugins/xml_sax/Makefile.am
--- gnumeric-0.72.orig/plugins/xml_sax/Makefile.am	Sun Oct  7 22:59:20 2001
+++ gnumeric-0.72/plugins/xml_sax/Makefile.am	Tue Oct 23 15:48:28 2001
@@ -13,6 +13,7 @@
 xml_DATA = $(xml_in_files:.xml.in=.xml)
 
 XML_I18N_XML_KIND = --pass-through
-@XML_I18N_MERGE_XML_RULE@
+@XML_I18N_MERGE_XML_RULE1@
+@XML_I18N_MERGE_XML_RULE2@
 
 EXTRA_DIST = $(xml_in_files) $(xml_DATA)
