diff -Nuard gnumeric-1.2.2.orig/gnumeric.mime gnumeric-1.2.2/gnumeric.mime
--- gnumeric-1.2.2.orig/gnumeric.mime	2003-11-03 07:55:35.000000000 +0100
+++ gnumeric-1.2.2/gnumeric.mime	2003-12-23 00:30:48.222278736 +0100
@@ -4,33 +4,6 @@
 application/vnd.ms-excel
 	ext: xls xlw xlt
 
-application/msexcel
-	ext: xls xlw xlt
-
-application/x-msexcel
-	ext: xls xlw xlt
-
-application/x-ms-excel
-	ext: xls xlw xlt
-
-application/vnd.ms-excel
-	ext: xls xlw xlt
-
-application/x-excel
-	ext: xls xlw xlt
-
-x-dos_ms_excel
-	ext: xls xlw xlt
-
-application/xls
-	ext: xls xlw xlt
-
-application/x-xls
-	ext: xls xlw xlt
-
-zz-winassoc-xls           
-	ext: xls xlw xlt
-
 application/x-applix-spreadsheet
 	ext: as 
 
