diff -Nru gnumeric-0.60/configure.in gnumeric-0.60.new/configure.in
--- gnumeric-0.60/configure.in	Fri Dec  8 15:41:13 2000
+++ gnumeric-0.60.new/configure.in	Fri Dec  8 15:42:03 2000
@@ -244,8 +244,8 @@
 gnumeric_prefix="${prefix}"
 gnumeric_exec_prefix="${exec_prefix}"
 gnumeric_version="${VERSION}${bonobo_suffix}"
-gnumeric_datadir="${datadir}/gnumeric/${gnumeric_version}"
-gnumeric_libdir="${libdir}/gnumeric/${gnumeric_version}"
+gnumeric_datadir="${datadir}/gnumeric/"
+gnumeric_libdir="${libdir}/gnumeric/"
 gnumeric_plugindir="${gnumeric_libdir}/plugins"
 gnumeric_gladedir="${gnumeric_datadir}/glade"
 gnumeric_autoformatdir="${gnumeric_datadir}/autoformat-templates"
diff -Nru gnumeric-0.60/src/dialogs/Makefile.am gnumeric-0.60.new/src/dialogs/Makefile.am
--- gnumeric-0.60/src/dialogs/Makefile.am	Fri Dec  8 15:41:13 2000
+++ gnumeric-0.60.new/src/dialogs/Makefile.am	Fri Dec  8 15:15:02 2000
@@ -10,7 +10,7 @@
 gnumeric_version="$(VERSION)"
 libdialogs_a_SOURCES =	$(std_SOURCES)
 endif
-gnumeric_libdir = $(datadir)/gnumeric/$(gnumeric_version)
+gnumeric_libdir = $(datadir)/gnumeric
 
 INCLUDES = 								\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\" 			\
--- gnumeric-0.71/src/dialogs/Makefile.am.orig	Tue Oct  9 02:51:39 2001
+++ gnumeric-0.71/src/dialogs/Makefile.am	Tue Oct  9 02:55:35 2001
@@ -4,10 +4,8 @@
 iconsdir = $(datadir)/pixmaps/gnumeric
 
 if BONOBO
-gnumeric_version="$(VERSION)-bonobo"
 libdialogs_a_SOURCES =	$(base_files) dialog-graph-guru.c
 else
-gnumeric_version="$(VERSION)"
 libdialogs_a_SOURCES =	$(base_files)
 endif
 gnumeric_libdir = $(datadir)/gnumeric
