--- gnumeric-0.59/po/POTFILES.in.org	Sat Nov 11 22:12:38 2000
+++ gnumeric-0.59/po/POTFILES.in	Sun Nov 19 00:58:24 2000
@@ -5,7 +5,6 @@
 plugins/excel/ms-excel-write.c
 plugins/excel/ms-formula-read.c
 plugins/gb/plugin.c
-plugins/gda/plugin-gda.c
 plugins/guile/plugin.c
 plugins/html/boot.c
 plugins/html/html.c
@@ -127,7 +127,6 @@
 src/sheet-object-container.c
 src/sheet-object-widget.c
 src/sheet-object.c
-src/sheet-view.c
 src/sheet.c
 src/solver-lp.c
 src/stf-export.c
