diff -Nur gnumeric-0.67.orig/Makefile.am gnumeric-0.67/Makefile.am
--- gnumeric-0.67.orig/Makefile.am	Thu Jun 28 22:51:29 2001
+++ gnumeric-0.67/Makefile.am	Wed Jul  4 15:46:22 2001
@@ -40,11 +40,6 @@
 	xml-i18n-update.in	\
 	$(oaf_in_files)
 
-if BONOBO
-gnumeric_version="$(VERSION)-bonobo"
-else
-gnumeric_version="$(VERSION)"
-endif
 
 ## Cloned from gnome-print
 ##
diff -Nur gnumeric-0.67.orig/configure.in gnumeric-0.67/configure.in
--- gnumeric-0.67.orig/configure.in	Thu Jun 28 22:53:00 2001
+++ gnumeric-0.67/configure.in	Wed Jul  4 15:46:49 2001
@@ -77,7 +77,7 @@
 dnl and why we have so many translations.
 dnl
 ALL_LINGUAS="ca cs da de en_AU en_CA en_GB el es es_DO es_GT es_HN es_MX es_PA es_PE es_SV et fi fr ga gl hr hu it ja nl no pl pt pt_BR ru sk sv tr uk zh_CN.GB2312 zh_TW.Big5"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 # AM_GNOME_GETTEXT above substs $DATADIRNAME
 # this is the directory where the *.{mo,gmo} files are installed
 gnomelocaledir='${prefix}/${DATADIRNAME}/locale'
diff -Nur gnumeric-0.67.orig/src/Makefile.in gnumeric-0.67/src/Makefile.in
--- gnumeric-0.67.orig/src/Makefile.in	Fri Jun 29 00:23:30 2001
+++ gnumeric-0.67/src/Makefile.in	Wed Jul  4 15:46:22 2001
@@ -27,6 +27,7 @@
 sbindir = @sbindir@
 libexecdir = @libexecdir@
 datadir = @datadir@
+gladedir = $(datadir)/gnumeric/glade
 sysconfdir = @sysconfdir@
 sharedstatedir = @sharedstatedir@
 localstatedir = @localstatedir@
diff -Nur gnumeric-0.67.orig/src/dialogs/Makefile.am gnumeric-0.67/src/dialogs/Makefile.am
--- gnumeric-0.67.orig/src/dialogs/Makefile.am	Sun Jun 17 06:55:26 2001
+++ gnumeric-0.67/src/dialogs/Makefile.am	Wed Jul  4 15:46:22 2001
@@ -4,10 +4,8 @@
 iconsdir = $(datadir)/pixmaps/gnumeric
 
 if BONOBO
-gnumeric_version="$(VERSION)-bonobo"
 libdialogs_a_SOURCES =	$(std_SOURCES) dialog-graph-guru.c
 else
-gnumeric_version="$(VERSION)"
 libdialogs_a_SOURCES =	$(std_SOURCES)
 endif
 gnumeric_libdir = $(datadir)/gnumeric/$(gnumeric_version)
diff -Nur gnumeric-0.67.orig/src/widgets/widget-editable-label.c gnumeric-0.67/src/widgets/widget-editable-label.c
--- gnumeric-0.67.orig/src/widgets/widget-editable-label.c	Tue Mar 13 02:00:52 2001
+++ gnumeric-0.67/src/widgets/widget-editable-label.c	Wed Jul  4 15:46:22 2001
@@ -124,6 +124,7 @@
 {
 	gtk_widget_grab_focus (GTK_WIDGET (el));
 
+	gtk_widget_grab_focus (GTK_WIDGET (el));
 	/*
 	 * Create a GtkEntry to actually do the editing.
 	 */
