--- gnumeric-1.12.45/configure.ac.orig	2018-11-24 18:24:59.000000000 +0100
+++ gnumeric-1.12.45/configure.ac	2019-07-04 19:58:36.996040340 +0200
@@ -302,12 +302,8 @@
 		[gda_msg=yes],
 		[gda_msg="NO.  libgda problem"])
 	if test "$gda_msg" = yes; then
-		PKG_CHECK_MODULES(GNOMEDB, [libgnomedb-4.0 >= 3.99.6],
-			[gnomedb_msg="yes"],
-			[gnomedb_msg="NO. libgnomedb problem"])
-		if test "$gnomedb_msg" = yes; then
-			AC_DEFINE(HAVE_LIBGNOMEDB, 1, [ Define if libgnomedb support is compiled in])
-		fi
+		gnomedb_msg="yes"
+		AC_DEFINE(HAVE_LIBGNOMEDB, 1, [ Define if libgnomedb support is compiled in])
 	fi
 fi
 AM_CONDITIONAL(WITH_GDA, test x"$gda_msg" = xyes)
--- gnumeric-1.12.45/plugins/gnome-db/plugin-gnomedb.c.orig	2018-05-16 22:23:05.000000000 +0200
+++ gnumeric-1.12.45/plugins/gnome-db/plugin-gnomedb.c	2019-07-04 20:08:16.359568325 +0200
@@ -15,12 +15,10 @@
 	argv[0] = (char *) "gnome-database-properties-4.0";
 	argv[1] = NULL;
 	if (!g_spawn_async (NULL, argv, NULL, G_SPAWN_SEARCH_PATH, NULL, NULL, NULL, NULL)) {
-		char *msg = g_strdup_printf (
+		go_gtk_notice_dialog (wbcg_toplevel (WBC_GTK (wbc)),
+			GTK_MESSAGE_INFO, 
 			_("Could not run GNOME database configuration tool ('%s')"),
 			argv[0]);
-		go_gtk_notice_dialog (wbcg_toplevel (WBC_GTK (wbc)),
-			GTK_MESSAGE_INFO,  msg);
-		g_free (msg);
 	}
 }
 
