diff -Nur gnumeric-0.71.orig/Makefile.am gnumeric-0.71/Makefile.am
--- gnumeric-0.71.orig/Makefile.am	Mon Oct  8 03:46:33 2001
+++ gnumeric-0.71/Makefile.am	Wed Oct 10 18:50:40 2001
@@ -46,11 +46,6 @@
 	$(oaf_in_files)		\
 	depcomp	# This should not be here, but whatever is supposed to add it, is not adding it, and I need to do a release.
 
-if BONOBO
-gnumeric_version="$(VERSION)-bonobo"
-else
-gnumeric_version="$(VERSION)"
-endif
 
 ## Cloned from gnome-print
 ##
