--- templates/autoformat/Makefile.am.orig	2005-01-20 10:20:23.267209728 +0100
+++ templates/autoformat/Makefile.am	2005-01-20 10:21:01.721363808 +0100
@@ -41,13 +41,13 @@
 		    < $(srcdir)/$$cat.category.in			\
 		    > $$dir/.category 	;				\
 		for f in $(srcdir)/autoformat.$$cat.*.xml.in ; do	\
-			echo "	$${f#$(srcdir)/autoformat.$$cat.}" ;	\
+			echo "  `echo $${f} | sed -e 's#.*autoformat.$$cat.##g'`"; \
 			sed 						\
 				-e 's/ _author *=/ author=/'		\
 				-e 's/ _name *=/ name=/' 		\
 				-e 's/ _description *=/ description=/'	\
 			    < $$f 					\
-			    > $$dir/`basename $${f%.in}` ;		\
+			    > $$dir/`basename `echo $${f} | sed -e 's#.xml.in#.xml#g'``;\
 		done							\
 	done
 
