--- gnumeric-0.35/src/widgets/widget-editable-label.c.sopwith	Mon Sep 13 17:44:22 1999
+++ gnumeric-0.35/src/widgets/widget-editable-label.c	Mon Sep 13 17:44:43 1999
@@ -120,6 +120,7 @@
 {
 	GtkWidget *toplevel;
 
+	gtk_widget_grab_focus (GTK_WIDGET (el));
 	/*
 	 * Create a GtkEntry to actually do the editing.
 	 */
--- gnumeric-0.35/src/Makefile.in.sopwith	Tue Sep  7 20:39:00 1999
+++ gnumeric-0.35/src/Makefile.in	Mon Sep 13 17:44:08 1999
@@ -42,6 +42,7 @@
 sbindir = @sbindir@
 libexecdir = @libexecdir@
 datadir = @datadir@
+gladedir = $(datadir)/gnumeric/glade
 sysconfdir = @sysconfdir@
 sharedstatedir = @sharedstatedir@
 localstatedir = @localstatedir@
