--- gnumeric-0.72/configure.in.orig	Tue Oct 23 16:01:13 2001
+++ gnumeric-0.72/configure.in	Tue Oct 23 16:26:55 2001
@@ -678,6 +678,9 @@
 AC_SUBST(GDA_LIBS)
 
 GNUMERIC_LIBS=`gnome-config --libs gnomeui print libglade $bonobo libole2 gal`
+if test -n "$bonobo" ; then
+	GNUMERIC_LIBS="$GNUMERIC_LIBS -lbonobox"
+fi
 GNUMERIC_CFLAGS=`gnome-config --cflags gnomeui print libglade $bonobo libole2 gal`
 AC_SUBST(GNUMERIC_LIBS)
 AC_SUBST(GNUMERIC_CFLAGS)
