--- src/wbcg-actions.c.orig	2005-01-20 11:17:28.768454384 +0100
+++ src/wbcg-actions.c	2005-01-20 11:20:26.250473016 +0100
@@ -854,8 +854,8 @@
 #warning "handle translations when we generate them"
 #ifndef G_OS_WIN32
 	argv[0] = (char *)"yelp";
-	argv[1] = g_build_filename (gnm_sys_data_dir (), "doc", "C",
-			"gnumeric.xml", NULL);
+	argv[1] = g_build_filename ("/usr/share/gnome/help/gnumeric",
+			"C/gnumeric.xml", NULL);
 #else
 	argv[0] = (char *)"hh";
 	argv[1] = g_build_filename (gnm_sys_data_dir (), "doc", "C",
