diff -urN gnumeric-1.0.1.org/Makefile.am gnumeric-1.0.1/Makefile.am
--- gnumeric-1.0.1.org/Makefile.am	Thu Jan 10 01:22:16 2002
+++ gnumeric-1.0.1/Makefile.am	Thu Jan 10 13:16:09 2002
@@ -45,11 +45,6 @@
 	xml-i18n-update.in	\
 	$(oaf_in_files)
 
-if BONOBO
-gnumeric_version="$(VERSION)-bonobo"
-else
-gnumeric_version="$(VERSION)"
-endif
 
 ## Cloned from gnome-print
 ##
diff -urN gnumeric-1.0.1.org/configure.in gnumeric-1.0.1/configure.in
--- gnumeric-1.0.1.org/configure.in	Thu Jan 10 01:22:16 2002
+++ gnumeric-1.0.1/configure.in	Thu Jan 10 13:17:14 2002
@@ -153,7 +153,7 @@
 dnl and why we have so many translations.
 dnl
 ALL_LINGUAS="ca cs da de en_AU en_CA en_GB el es et fi fr ga gl hr hu it ja ko nl nn no pl pt pt_BR ro ru sk sv tr uk zh_CN zh_TW"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 # AM_GNOME_GETTEXT above substs $DATADIRNAME
 # this is the directory where the *.{mo,gmo} files are installed
 gnomelocaledir='${prefix}/${DATADIRNAME}/locale'
diff -urN gnumeric-1.0.1.org/doc/sgmldocs.make gnumeric-1.0.1/doc/sgmldocs.make
--- gnumeric-1.0.1.org/doc/sgmldocs.make	Thu Jan 10 01:22:21 2002
+++ gnumeric-1.0.1/doc/sgmldocs.make	Thu Jan 10 13:16:09 2002
@@ -48,7 +48,7 @@
 	-cp $(docname)/index.html .
 
 $(docname).sgml: $(sgml_ents)
-        -ourdir=`pwd`;  \
+        ourdir=`pwd`;  \
         cd $(srcdir);   \
         cp $(sgml_ents) $$ourdir
 
