diff -urN gnumeric-1.0.1.org/configure.in gnumeric-1.0.1/configure.in
--- gnumeric-1.0.1.org/configure.in	Thu Jan 10 01:22:16 2002
+++ gnumeric-1.0.1/configure.in	Thu Jan 10 13:17:14 2002
@@ -444,11 +444,11 @@
 )
 
 if $try_gb; then
-	AC_MSG_CHECKING(for Gb == 0.0.17						)
+	AC_MSG_CHECKING(for Gb >= 0.0.17						)
 	if gnome-config --libs gb > /dev/null 2>&1; then
 		verstxt=`gnome-config --modversion gb 2>/dev/null`
 		vers=`echo "$verstxt" | awk -F. '{ printf "%d", $1 * 10000 + $2 * 100 + $3; }'`
-		if test "$vers" -eq 000017; then
+		if test "$vers" -ge 000017; then
 			AC_MSG_RESULT($marker_ok found $verstxt)
 			AC_DEFINE(ENABLE_GB)
 			GB_LIBS=`gnome-config --libs gb`
diff -urN gnumeric-1.0.1.org/plugins/gb/Makefile.am gnumeric-1.0.1/plugins/gb/Makefile.am
--- gnumeric-1.0.1.org/plugins/gb/Makefile.am	Thu Jan 10 01:22:21 2002
+++ gnumeric-1.0.1/plugins/gb/Makefile.am	Thu Jan 10 16:44:07 2002
@@ -34,7 +34,7 @@
 gnum_gb_la_LIBADD =		\
 	$(GB_LIBS)		\
 	$(GNUMERIC_LIBS)	\
-	-lm @GLIB_LIBS@ -lgb -lgbrun
+	-lm @GLIB_LIBS@
 
 xml_in_files = plugin.xml.in
 xml_DATA = $(xml_in_files:.xml.in=.xml)
