--- gnumeric-1.0.9/configure.in~	Thu Sep 19 01:41:38 2002
+++ gnumeric-1.0.9/configure.in	Thu Sep 19 01:41:38 2002
@@ -112,14 +112,8 @@
 fi
 
 AM_CONDITIONAL(HAVE_GUILE, $build_guile)
+GNOME_CHECK_GUILE
 
-if $build_guile; then
-	GNOME_CHECK_GUILE
-fi
-
-AC_SUBST(CFLAGS)
-AC_SUBST(CPPFLAGS)
-AC_SUBST(LDFLAGS)
 
 dnl **************************************************
 dnl * psiconv support
@@ -714,7 +714,6 @@
 src/portability.h
 icons/Makefile
 idl/Makefile
-src/Makefile
 src/dialogs/Makefile
 src/widgets/Makefile
 src/functions/Makefile
